HotelManagement/src/main/java/app/controller/LoginController.java

```diff
@@ -0,0 +1,18 @@
+package app.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class LoginController {
+    
+    
+    
+    @GetMapping("/login")
+    public ModelAndView showLoginPage() {
+        return new ModelAndView("views/login/Login");
```

Comment:
```suggestion
        return new ModelAndView("views/sessions/login");
```
mình đổi tên folder ở view giúp chị nhé